arch/cpu/nrf/nrf-def.h

```diff
@@ -77,21 +84,10 @@
 #define GPIO_HAL_CONF_ARCH_HDR_PATH          "gpio-hal-arch.h"
 #define GPIO_HAL_CONF_ARCH_SW_TOGGLE         0
 /*---------------------------------------------------------------------------*/
-#ifndef TSCH_CONF_HW_FRAME_FILTERING
-#define TSCH_CONF_HW_FRAME_FILTERING  0
-#endif /* TSCH_CONF_HW_FRAME_FILTERING */
-
-#ifndef TSCH_CONF_RADIO_ON_DURING_TIMESLOT
-#define TSCH_CONF_RADIO_ON_DURING_TIMESLOT 1
-#endif /* TSCH_CONF_RADIO_ON_DURING_TIMESLOT */
-
-#ifndef TSCH_CONF_RESYNC_WITH_SFD_TIMESTAMPS
-#define TSCH_CONF_RESYNC_WITH_SFD_TIMESTAMPS 1
-#endif /* TSCH_CONF_RESYNC_WITH_SFD_TIMESTAMPS */
-
-#ifndef TSCH_CONF_TIMESYNC_REMOVE_JITTER
-#define TSCH_CONF_TIMESYNC_REMOVE_JITTER 0
-#endif /* TSCH_CONF_TIMESYNC_REMOVE_JITTER */
+#define TSCH_CONF_HW_FRAME_FILTERING          0
```

Comment:
What's the reason for removing the ifndef checks?

Reply:
Thanks for the catch. I don't know why I did this.

Reply:
Reverted